src/OpenTelemetry/Metrics/PeriodicExportingMetricReaderOptions.cs

```diff
@@ -14,10 +14,37 @@
 // limitations under the License.
 // </copyright>
 
+using Microsoft.Extensions.Configuration;
+using OpenTelemetry.Internal;
+
 namespace OpenTelemetry.Metrics
 {
     public class PeriodicExportingMetricReaderOptions
     {
+        internal const string OTelMetricExportIntervalEnvVarKey = "OTEL_METRIC_EXPORT_INTERVAL";
+        internal const string OTelMetricExportTimeoutEnvVarKey = "OTEL_METRIC_EXPORT_TIMEOUT";
```

Comment:
I've pondered whether we can push all environment variable processing into the `SdkOptions` class. Maybe this wouldn't play well in the case where you have multiple periodic readers each with their own distinct set of options...

Reply:
Maybe nevermind... I just groked that this got moved to the options class. I think this makes sense cause now whenever a new `PeriodicExportingMetricReaderOptions` is instantiated it's initialized from the environment variables.